samigo/samigo-app/src/webapp/css/tool_sam.css

```diff
@@ -1444,6 +1444,10 @@ input[type='reset'].linearButton {
 	padding: 6px 12px;
 }
 
+.MathJax_Display {
+  text-align: left !important;
```

Comment:
Lets not use !important

Reply:
Important is needed because overrides the MathJax default CSS injected via JS.